keyboards/handwired/promethium/keymaps/priyadi/keymap.c

```diff
@@ -1277,7 +1279,9 @@ void set_output_user(uint8_t output) {
 }
 
 void matrix_init_user() {
+#ifdef __AVR__
   _delay_ms(500); // give time for usb to initialize
```

Comment:
I believe that if you change ` _delay_ms(500)` to `wait_ms(500)`, you can remove the `ifdef` check. My understanding is that `_delay_ms()` only works on AVR, but `wait_ms()` is platform-agnostic.

Reply:
Yup. @noroadsleft is 100% correct.

`wait_ms` is actually a macro/alias for the platform specific code, so that it's essential hardware agnostic. 

https://github.com/qmk/qmk_firmware/blob/6e984a8b5e34ba181b0893a929af569a1faef2b6/tmk_core/common/wait.h#L10-L27

Reply:
thank you for your suggestion. yes, changing it to wait_ms works.